include/asm/symbol.h

```diff
@@ -127,6 +127,8 @@ struct Symbol *sym_FindExactSymbol(char const *symName);
 struct Symbol *sym_FindUnscopedSymbol(char const *symName);
 // Find a symbol, possibly scoped, by name
 struct Symbol *sym_FindScopedSymbol(char const *symName);
```

Comment:
When is that one still used, then?

Reply:
When the call site, or some grandparent, does its own handling of PC or NARG.